### ISSUE TITLE
MozSciLab countdown

### ISSUE THREAD

**Issue by betatim** [everware/jupyterhub#17]
Things we need to do before the mozilla science lab blog post. :red_circle: = critical, needs to be done before, :green_heart: = done, :large_orange_diamond: = nice to have
- [ ] :red_circle: validate user input so that both `https://github.com/...` and `git@github.com/...` work as repo URLs
- [ ] :red_circle: make sure logout button is always visible to escape `error 500` 
- [ ] :red_circle: use github oauth so we have names of those abusing the service

**Comment by betatim** [comment]
- [ ] :red_circle: add "beta" to the "everware" name on the landing page
- [ ] :red_circle: make sure all examples run
  - [ ] fix dimuon example
  - [ ] fix roofit-tutorial example
- [ ] :large_orange_diamond: link back to github organisation from everware pages
- [ ] :large_orange_diamond: everware needs a logo

**Comment by anaderi** [comment]
- [ ] :large_orange_diamond: kill idle containers

**Comment by betatim** [comment]
This is now being worked on in everware/everware#5
